### PR TITLE
Write last dir on quit in vim picker mode

### PR DESCRIPTION
**Pull request by mroavi**
Resolves https://github.com/jarun/nnn/issues/689
Needed to implement https://github.com/mcchrish/nnn.vim/issues/57

### CODE REVIEW

File: src/nnn.c [hunk 1]
```diff
@@ -6615,10 +6615,11 @@ static bool browse(char *ipath, const char *session, int pkey)
 				save_session(TRUE, NULL);
 
 			/* CD on Quit */
-			/* In vim picker mode, clear selection and exit */
-			/* Picker mode: reset buffer or clear file */
-			if (sel == SEL_QUITCD || getenv("NNN_TMPFILE"))
-				g_state.picker ? selbufpos = 0 : write_lastdir(path);
+			if (sel == SEL_QUITCD || getenv("NNN_TMPFILE")) {
+				write_lastdir(path);
+				if(g_state.picker)
```
Comment (jarun) [inline src/nnn.c:6620]: Space after `if`.

Comment (jarun) [reply src/nnn.c:6620]: If I wasn't clear, I wanted to say you should have space after the `if` token, not the other way. That's what we follow in `nnn`.

Comment (mroavi) [reply src/nnn.c]: My bad.

**Comment by jarun** [comment]
Thank you!

**Comment by jarun** [comment]
Please update your master or re-fork. I removed the additional to commits about `if` to keep the commit history clean.

**Comment by jarun** [comment]
@mroavi can you help me get this working? https://github.com/mcchrish/nnn.vim#action

I have raised https://github.com/mcchrish/nnn.vim/issues/71 but still waiting for a response.

**Pull request merged**